code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -121,9 +121,9 @@ GLOBAL_DATUM_INIT(canister_icon_container, /datum/canister_icons, new())
 
 /obj/machinery/atmospherics/portable/canister/examine(mob/user)
 	. = ..()
-	. += "<span class='notice'>The canister can be connected to a connector port with a wrench. Tanks of gas (the kind you can hold in your hand) \
-			can be filled by the canister, by using the tank on the canister, increasing the release pressure, then opening the valve until it is full, and then close it. \
-			*DO NOT* remove the tank until the valve is closed. A gas analyzer can be used to check the contents of the canister.</span>"
+	. += "<span class='notice'>Connect a canister to a connector port using a wrench. To fill a tank, attach it to the caniser, increase the \
+			release pressure, and open the valve. Alt-click to eject the tank, or use another to hot-swap. A gas analyzer can be used to check \
+			the contents of the canister.</span>"
 	if(isAntag(user))
 		. += "<span class='notice'>Canisters can be damaged, spilling their contents into the air, or you can just leave the release valve open.</span>"
```

Comment:
It has examine text with tips for antgs????

Reply:
Yep, this was carried over in detailed examine removal. 
Is antag also checks for special role, NOT actually being an antag, ERT and stuff gets this, same as it was

Reply:
Several things do have a custom antag examine yes. Suprised me as well.